File: java/visitor/ScheduleManagementVisitor.java
package visitor;

public interface ScheduleManagementVisitor {
    void visit(DayShiftScheduleManagement dayShiftScheduleManagement);
    void visit(NightShiftScheduleManagement nightShiftScheduleManagement);
    void visit(RemoteWorkShiftScheduleManagement remoteWorkShiftScheduleManagement);
}

File: PHP/Patterns/src/Behavioral/Visitor/Visitors/ScheduleManagementVisitor.php
<?php

namespace src\Behavioral\Visitor\Visitors;

use src\Behavioral\Visitor\DayShiftScheduleManagement;
use src\Behavioral\Visitor\NightShiftScheduleManagement;
use src\Behavioral\Visitor\RemoteWorkShiftScheduleManagement;

interface ScheduleManagementVisitor
{
    function visitDayShift(DayShiftScheduleManagement $dayShiftScheduleManagement);
    function visitNightShift(NightShiftScheduleManagement $nightShiftScheduleManagement);
    function visitRemoteWorkShift(RemoteWorkShiftScheduleManagement $remoteWorkShiftScheduleManagement);
}